.circleci/config.yml

```
@@ -36,6 +36,31 @@ aliases:
       command: |
         sudo apt-get update
         sudo apt-get install -y libcairo2-dev
+  - &check_versions
+    run:
+      name: Check that all versions are the same if $CIRCLE_TAG is defined
+      command: |
+        if [[ ! -z ${CIRCLE_TAG+x} ]]; then
+          python -m pip install --upgrade setuptools_scm
+          BACKEND_VERSION=$(python -m setuptools_scm)
+          echo "richie version: ${BACKEND_VERSION}"
+          # In the frontend
+          cat src/frontend/package.json | grep "\"version\": \"${BACKEND_VERSION}\",$"
```

```suggestion
          grep "\"version\": \"${BACKEND_VERSION}\",$" src/frontend/package.json
```
Only for nitpicking…